cmd/minishift/cmd/start.go

```diff
@@ -261,6 +261,13 @@ func postClusterUp(machineName string, ip string, port int, routingSuffix string
 		atexit.Exit(1)
 	}
 
+	// Run the default addons if MINISHIFT_HOME dir does not exist or empty
+	if !homeDirExists || util.IsDirEmpty(constants.Minipath) {
```

Comment:
@hferentschik , I though of delaying at later section as we might have some addons in future which might have requirement of VM in running state.

I am fine with either way.